### PR TITLE
Updated with info about adding new folders

### PR DESCRIPTION
**Pull request by Vium**
I found the folder id field a little bit confusing at first glance and I couldn't find anything in the docs. perhaps this helps.
Added to FAQ first, moved over here as suggested by @imsodin

### CODE REVIEW

File: intro/gui.rst [hunk 1]
```diff
@@ -50,6 +50,20 @@ Error
 	prevent data loss e.g. when the folder path was unmounted. If the marker
 	was deleted accidentally, just recreate it and press the rescan button
 	in the UI.
+	
+Creating new synced Folders
```
Comment (AudriusButkevicius) [inline intro/gui.rst]: Is there a reason Folders is capital and plural?

File: intro/gui.rst [hunk 2]
```diff
@@ -50,6 +50,20 @@ Error
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the Syncthing web interface. Click the "Add Folder" button:
```
Comment (AudriusButkevicius) [inline intro/gui.rst]: Click the "Add Folder" button which will bring up bla bla bla.

File: intro/gui.rst [hunk 3]
```diff
@@ -50,6 +50,20 @@ Error
...
+
+Set the folder label to something descriptive. The label you enter here will be shared with remote devices, but can be changed on each device, as desired.
```
Comment (AudriusButkevicius) [inline intro/gui.rst]: et the folder label to something descriptive as the .... This label can be...

File: intro/gui.rst [hunk 4]
```diff
@@ -50,6 +50,20 @@ Error
...
+
+The folder id has to be the same across all devices, it is the unique identifier of the folder. Best practice is to keep the auto-generated id to avoid conflicts with equally named folders on other devices.
```
Comment (AudriusButkevicius) [inline intro/gui.rst]: all devices, as it is...

File: intro/gui.rst [hunk 5]
```diff
@@ -50,6 +50,20 @@ Error
...
+
+The folder path designates the physical path to the folder, i.e. on your hard drive.
+
+Next select with which devices to share and you are good to go.
```
Comment (AudriusButkevicius) [inline intro/gui.rst]: to share the folder with and press add, which will add the folder and start syncing.

**Comment by Vium** [comment]
...

**Comment by imsodin** [comment]
@st-review merge

intro/gui: Add info about folder creation

**Comment by st-review** [comment]
:ok_hand: Merged as e37e210e68b1df53ebe6bed997ea56557cf877fe. Thanks, @Vium!

**Pull request closed without merging**